frontend/public/components/operator-lifecycle-manager/descriptors/spec/index.tsx

```diff
@@ -74,6 +77,19 @@ const BooleanSwitch: React.FC<SpecCapabilityProps> = (props) => {
   </div>;
 };
 
+const Secret: React.FC<SpecCapabilityProps> = (props) => {
+  const [reveal, setReveal] = React.useState(false);
+
+  return <React.Fragment>
+    <button className="btn btn-link" type="button" onClick={() => setReveal(!reveal)}>
+      {reveal
+        ? <React.Fragment><EyeSlashIcon className="co-icon-space-r" />Hide Values</React.Fragment>
+        : <React.Fragment><EyeIcon className="co-icon-space-r" />Reveal Values</React.Fragment>}
+    </button>
+    <SecretValue value={Base64.encode(props.value)} reveal={reveal} />
```

Comment:
We might rework the `SecretValue` component so you don't have to artificially base64 encode the value, but ok for now

Reply:
Maybe an optional prop saying whether the secret value is a base64 encoded

Reply:
Switched the `<SecretValue>` component to accept `props.encoded`.